src/lib/gui/config/Screen.cpp

```diff
@@ -90,12 +90,13 @@ QString Screen::screensSection() const
   for (int i = 0; i < fixes().size(); i++)
     out.append(lineTemplate.arg(fixName(i), fixes().at(i) ? QStringLiteral("true") : QStringLiteral("false")));
 
-  out.append(QStringLiteral("\t\tswitchCorners = none"));
-  for (int i = 0; i < switchCorners().size(); i++)
+  auto corners = QStringLiteral("none");
+  for (int i = 0; i < switchCorners().size(); i++) {
     if (switchCorners()[i])
-      out.append(QStringLiteral("+%1 ").arg(switchCornerName(i)));
+      corners.append(QStringLiteral(" +%1 ").arg(switchCornerName(i)));
```

Comment:
The extra space char at the end doesn't seem to be needed.
```suggestion
      corners.append(QStringLiteral(" +%1").arg(switchCornerName(i)));
```

Reply:
This fixes the parsing .... without it crashes...

Reply:
I don't think so. It appears the fix is to move the space from right to left. The extra space at the end isn't really needed. I tried with just the left space and it works fine.

Reply:
Decision on Matrix: Leave the extra space in and @sithlord48 will remove it later.